src/main/webapp/i18n/de/importUsers.json

```diff
@@ -8,7 +8,7 @@
             "csvExampleText": "Beispiel für eine gültige CSV Datei:",
             "csvFile": {
                 "label": ".csv Datei auswählen",
-                "tooltip": "Bitte wähle die .csv Datei mit den Nutzern zum Import aus. Pro Eintrag muss entweder die 'LOGIN', 'REGISTRATION_NUMBER' oder 'EMAIL' Spalte ausgefüllt sein. Folgendes Format muss dabei beachtet werden: Login, Matrikelnummer, E-Mail, Vorname, Nachname. Du musst diese in der obersten Spalte angeben!"
+                "tooltip": "Bitte wähle die .csv Datei mit den Nutzern zum Import aus. Pro Eintrag muss entweder die 'LOGIN', 'REGISTRATION_NUMBER' oder 'E-MAIL' Spalte ausgefüllt sein. Folgendes Format muss dabei beachtet werden: Login, Matrikelnummer, E-Mail, Vorname, Nachname. Du musst diese in der obersten Spalte angeben!"
```

Comment:
If anyone else wonders, the import accepts both versions with and without hyphen